lib/pbench/server/cache_manager.py

```diff
@@ -266,11 +267,13 @@ def create(cls, tarball: Path, controller: "Controller") -> "Tarball":
 
         return cls(destination, controller)
 
-    def extract(self, path: str) -> str:
+    @staticmethod
+    def extract(tarball_path: str, path: str) -> str:
```

Comment:
Not sure why you made this change. As an instance method, it has direct access to the tarball path already?

Reply:
And, the LGTM bot seems to [have a concern](https://github.com/distributed-system-analysis/pbench/pull/3105#issuecomment-1339960714) about [this change](https://lgtm.com/projects/g/distributed-system-analysis/pbench/snapshot/4e129343f0c4d5f0ea51af8cf39deb71ff66e048/files/lib/pbench/server/cache_manager.py?sort=name&dir=ASC&mode=heatmap#xab785682da398daa:1), too.  😁